apps/gitness/src/pages/profile-settings/profile-settings-theme-container.tsx

```diff
@@ -0,0 +1,9 @@
+import { SettingsAccountThemePage } from './profile-settings-theme-page'
+
+export const SettingsProfileThemePage = () => {
+  return (
+    <>
+      <SettingsAccountThemePage />
+    </>
+  )
+}
```

Comment:
if we're not making API calls lets directly render the page? Do we need a container?

Reply:
Removed